inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -498,15 +499,19 @@ def refextract(obj, eng):
         None
 
     """
+    source = LiteratureReader(obj.data).source
+    journal_kb_dict = create_journal_kb_dict()
+
     if 'references' in obj.data:
-        extracted_raw_references = dedupe_list(extract_references_from_raw_refs(obj.data['references']))
+        if current_app.config.get("FEATURE_FLAG_ENABLE_REFEXTRACT_SERVICE"):
+            extracted_raw_references = dedupe_list(extract_references_from_reference_list(obj.data['references'], custom_kbs_file=journal_kb_dict, source=source))
```

Comment:
Here you are calling directly the service, there is some work that needs to be done before you do this. Take a look into that part 
https://github.com/inspirehep/inspire-next/blob/6400e115a2ab958e92e27a11652df32af1c429c0/inspirehep/modules/workflows/tasks/refextract.py#L251-L307